mne/decoding/transformer.py

```diff
@@ -114,8 +114,14 @@ def fit(self, epochs_data, y):
                 scaler = clone(std_scaler)
                 scaler.fit(X[pick].T)
                 self.scalers_.append(scaler)
-                self.ch_mean_[pick], self.scale_[pick] = (
-                    scaler.mean_, scaler.scale_)
+
+                # StandardScaler scale_ attribute introduced in version 0.18
+                try:
+                    self.ch_mean_[pick], self.scale_[pick] = (
+                        scaler.mean_, scaler.scale_)
+                except:
+                    self.ch_mean_[pick], self.scale_[pick] = (
+                        scaler.mean_, scaler.std_)
```

Comment:
I wasn't sure what @kingjr meant by the deprecation. I thought he was asking me because it was some code which I had written (=legacy code). But looks like this part of the code is completely new ...